.gitignore

```
@@ -86,9 +86,14 @@ site
 
 # Local data files
 **/datafile/*
+# Persistent storage should not be stored in git except for the markdown files
+persistent_storage
+!persistent_storage/**/*.md
 
 # Python Django
 *.log
 local_settings.py
 db.sqlite3
 db.sqlite3-journal
+# We'll disable migrations for now, until we have actual users
+sprout/migrations
```

```suggestion
migrations
```

Ignore any migration files, rather than just in `sprout/` folder.